### ISSUE TITLE
Form field should be labelled in some way

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#706]
# a11y bug
```
Erro: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.                                                                                                   
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #searchModal > div > div > div:nth-child(2) > div > form > div > div:nth-child(1) > input
   └── <input type="text" class="form-control" placeholder="Search Platforms, software or administrations">
```

**Comment by Trustidiong** [comment]
Hey, I would like to work on this issue. I think it will be fun.

**Comment by mishrajiharsh219** [comment]
Addressing #706
Hey, I would love to work on this issue, I will be able to solve this.

**Comment by CilgaIscan** [comment]
I would like to work on this. Could you assign this to me ? @libremente

**Comment by libremente** [comment]
Thanks everyone for your interest. @CilgaIscan you can work on this one! 😃

**Comment by bfabio** [comment]
Hi @CilgaIscan, are you still working on this one?
